flink-yarn/src/main/java/org/apache/flink/yarn/configuration/YarnConfigOptions.java

```diff
@@ -296,6 +296,14 @@
                     .noDefaultValue()
                     .withDescription("Specify YARN node label for the YARN application.");
 
+    public static final ConfigOption<String> TASK_MANAGER_NODE_LABEL =
+            key("yarn.taskmanager.node-label")
+                    .stringType()
+                    .noDefaultValue()
+                    .withDescription(
+                            "Specify YARN node label for the Flink TaskManagers, "
+                                    + "it will override the yarn.application.node-label if both are set.");
```

Comment:
nit: From my understanding, it will only override the node label for TMs. It would be good to clarify it.